specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2019-05-05-preview/AlertsManagement.json

```diff
@@ -52,6 +52,36 @@
         }
       }
     },
+    "/providers/Microsoft.AlertsManagement/alertsMetaData": {
```

Comment:
alertsMetadata is very generic name for this operation. Can you make it more specific. Like monitorProviders. What other things will you add to this meta data? If its not going to be lot of new things, may be you can have specific APIs to surface that information instead of clubbing it all in this API.

Reply:
We wanted a generic name only. Since we want to use this resource type for utility APIs. I mean we're going to return some or other meta data info with this resource type API based on identifier passed in API param